components/shopify_developer_app/README.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+By creating a custom app on Shopify, you will be able to configure the exact scopes that you require to build the workflows that you need.
+
+# Getting Started
+
+To get started, you will need to create a custom Shopify app in the Shopify Admin UI. The steps are outlined below:
+
+## Create a Shopify app
+1. Sign in to your Shopify admin account, and navigate to the store that you are managing. 
+2. Click **Develop apps**.
+3. Click **Allow custom app development**.
+4. Click **Create an App**, and name the app "Pipedream"
+
+  ![Create an App](https://res.cloudinary.com/dpenc2lit/image/upload/v1688060015/Screenshot_2023-06-29_at_10.11.43_AM_unkom4.png)
+
+## Configure Admin API scopes
+1. In the new app you have created, under the **Configuration** tab, click **Configure Admin API scopes**
+2. Select the scopes that you require for your use case. You may modify your scopes at any time by returning to 
+
+## Generate the Admin API access token
+1. Under API credentials, click **Install app**.
+2. Click **Reveal token once** and save it in a secure location (we recommend using a password manager such as 1Password) -- you will need it when setting up authentication on Pipedream, and it is only revealed once. If you happen to lose this, you will need to uninstall the app, and reinstall it on Shopify to generate a new access token.
+
+  ![API Credentials](https://res.cloudinary.com/dpenc2lit/image/upload/v1688061470/Screenshot_2023-06-29_at_10.54.53_AM_jta5gc.png)
+
+## Connect your Shopify app Pipedream with using your access token
+
+At this point, you should have a Pipedream App connected to your Shopify store, and a long-lived access token.
```

Comment:
asked in Slack, but why are we using long lived access tokens instead of oauth here?

Reply:
Resolved in Slack, leaving comment for posterity:
tl;dr, Shopify uses different routes of authentication depending on how the store was created:

![](https://cdn.shopify.com/shopifycloud/shopify_dev/assets/apps/auth-decision-tree-no-types-4bf1a26855f63f178107c8323b2b23d8c7fdcde632618dc939bd0d1ebb41e190.png)